app/(root)/questions/[year]/[day]/page.tsx

```
@@ -0,0 +1,33 @@
+  if (!questionData) {
+    return <div className="text-red px-2">Question not found.</div>;
+  }
+
+  return (
+    <main className="p-4">
+      <h1 className="text-lg">
+        --- Day: {day} - {questionData.title || `${year} - ${day}`} ---
+      </h1>
+      <article
+        className={"font-thin"}
+        dangerouslySetInnerHTML={{ __html: questionData.contentHtml }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)